tests/zap_helper_test.py

```diff
@@ -518,6 +518,28 @@ def test_load_session_error(self, session_mock, isfile_mock):
         with self.assertRaises(ZAPError):
             self.zap_helper.load_session(file_path)
 
+    @patch('zapv2.core.xmlreport')
+    @patch('zapcli.zap_helper.ZAPHelper.write_report')
```

Comment:
Hmm, the `xml_report` and `html_report` methods call directly the `write_report` method so I thank it make sense to test this assertion.. I can add a test for the `write_report` where I will use `mock_open`?